src/pages/Components/TextInput/index.md

```diff
@@ -7,52 +7,53 @@ order: 2
 
 > Inputs are used to **create input fields with text on a single line**. Their states depends on the user interaction or the context.
 
-<preview path="src/pages/Components/TextInput/previews/input-preview"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-preview"></preview>
 
 ## Standard input states :
 
 ### Hover
 
 When the input is hovered due to user navigation.
 
-<preview path="src/pages/Components/TextInput/previews/input-hover-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-hover-state"></preview>
 
 ### Focus
 
 When the input is in focus. It is generally triggered when the user clicks or taps on an element or selects it with the keyboard's "tab" key.
 
-<preview path="src/pages/Components/TextInput/previews/input-focus-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-focus-state"></preview>
 
 ### Disabled
 
 When the input field is inactive or inaccessible. It is usually used when the input, for some reason, is forbidden or blocked to be edited.
 
-<preview path="src/pages/Components/TextInput/previews/input-disabled-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-disabled-state"></preview>
 
 ### Valid
 
 When the entry is correct against the criteria determined upstream.
 
-<preview path="src/pages/Components/TextInput/previews/input-valid-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-valid-state"></preview>
 
 ### Invalid
 
 When the entry is invalid against the criteria determined upstream.
 
-<preview path="src/pages/Components/TextInput/previews/input-invalid-state"></preview>
+<preview nude="true" path="src/pages/Components/TextInput/previews/input-invalid-state"></preview>
 
-## Do's and don'ts
+## Using a left Icon :
 
-<hintitem>
-You can cumulate différents states like that :
-</hintitem>
-<preview path="src/pages/Components/TextInput/previews/input-invalid-event"></preview>
+Use a left icon to inform about a specific information type or format you expect the user to provide.
```

Comment:
i don't understand the term "format" ? Can we not just speak about action or feature?

Reply:
formatting would be better